app/server.rb

```diff
@@ -0,0 +1,24 @@
+require 'sinatra'
+require 'data_mapper'
+require 'rack-flash'
+require 'sinatra/partial'
+
+DataMapper::Logger.new($stdout, :debug)
+
```

Comment:
having the logger here is quite annoying - use it for specific debugging, then switch it off or set it to output to a file instead of stdout